tests/test_storage.py

```diff
@@ -329,7 +353,10 @@ def test_set_dispatch_plan(mock_market_config, storage_unit):
     storage_unit.outputs["soc"][start] = 0.5 * storage_unit.max_soc
 
     bids = strategy.calculate_bids(storage_unit, mc, product_tuples=product_tuples)
-    assert len(bids) == 0
+    assert len(bids) == 1
```

Comment:
This does actually change - why?

Reply:
It changed due to the removal of clearing empty bids.
It is readded and should be fine.